### ISSUE TITLE
Update photo_view dependency to 0.14.0

### ISSUE THREAD

**Issue by emilieroberts** [schibsted/tweet_ui#121]
**Describe the bug**
Flutter 3.10.6 assumes photo_view `0.14.0`. tweet_ui requires `0.13.0`, but this has an incompatible API.

Specifically, when you try to compile with 0.13.0 the error :
```
../../../../.pub-cache/hosted/pub.dev/photo_view-0.13.0/lib/src/core/photo_view_gesture_detector.dart:89:38: Error: No named parameter with the name 'kind'.
  }) : super(debugOwner: debugOwner, kind: kind);
                                     ^^^^
```

is thrown.

Can we update the photo_view dependency to `0.14.0`?

**Comment by wszeborowskimateusz** [comment]
https://github.com/schibsted/tweet_ui/pull/122

**Comment by wszeborowskimateusz** [comment]
@emilieroberts I released version 4.0.5. Hope this helps.

**Comment by emilieroberts** [comment]
Holy  toledo that was quick! Confirmed that updating to 4.0.5 now works for my app without having to override that dependency.

Thank you!
